server/lib/csp/blocking.js

```diff
@@ -19,12 +19,21 @@ function getOrigin(link) {
  * to be blocked if it runs afowl of a rule.
  */
 module.exports = function (config) {
-  var BLOB = 'blob:';
-  var CDN_URL = config.get('static_resource_url');
-  var DATA = 'data:';
-  var GRAVATAR = 'https://secure.gravatar.com';
-  var PUBLIC_URL = config.get('public_url');
-  var SELF = "'self'";
+  const AUTH_SERVER = getOrigin(config.get('fxaccount_url'));
+  const BLOB = 'blob:';
+  const CDN_URL = config.get('static_resource_url');
+  const DATA = 'data:';
+  // The sha of the embedded <style> tag in default-profile.svg.
+  const EMBEDDED_STYLE_SHA = "'sha256-9n6ek6ecEYlqel7uDyKLy6fdGNo3vw/uScXSq9ooQlk='";
+  const GRAVATAR = 'https://secure.gravatar.com';
+  const MARKETING_EMAIL_SERVER = getOrigin(config.get('marketing_email.api_url'));
+  const NONE = 'none';
```

Comment:
Excellent catch @jrgm, I didn't catch this because i have security errors turned off in the dev console. :/